### ISSUE TITLE
Community editing for instance admins

### ISSUE THREAD

**Issue by jmcharter** [LemmyNet/lemmy-ui#2138]
### Requirements

- [X] This is a feature request and not a bug report. Otherwise, please create a new [bug report](https://github.com/LemmyNet/lemmy-ui/issues/new?assignees=&labels=bug%2Ctriage&projects=&template=BUG_REPORT.yml) instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this request (or a similar one) already exists.
- [X] It's a single feature. Please don't request multiple features in one issue.

### Describe the feature you'd like

Currently, only moderators of a community are given the options to edit the community. However, if you end up with a rogue mod or one who has disappeared, then there's currently no way for an admin to make changes.

It would be good to have these options enabled for admins too.

**Comment by dessalines** [comment]
We don't let admins edit user-created data, other than to remove it.

**Comment by jmcharter** [comment]
@dessalines could I push you to expand on that a little?

My understanding of the role of an administrator is to take care of an entire instance. 

As it stands, an instance could have rogue moderators, ToS or law breaking descriptions and rules, etc. 

To me then, it makes sense that as an administrator of an instance that I should be able to intervene.

As it stands, my only option would be to purge a community and recreate it - removing all of the history of the community. 

The only alternative would be to make changes to the database directly. 

Given the latter is already possible, why not allow administrators to do this via the API? At least that way it can be logged and seen by everyone. 

Alternatively, and this could be made into a separate issue, allowing an admin to appoint themselves as moderator and transfer the community to themselves would achieve the same ultimate result. What do you think?

(Given that tone is missed from text, please note that my response is intended as a good faith discussion and not one of spite that this was dismissed).

**Comment by dessalines** [comment]
Because people including me, don't want admins to have the power to edit user data. It ruins content integrity. 

Admins can already remove or purge offending content, editing it isn't necessary.

**Comment by jmcharter** [comment]
What about offending content in the sidebar?
To be clear, I'm not asking for the ability to edit anybody's posts.

**Comment by dessalines** [comment]
Then either remove the community, or message the community creator about it.

**Comment by jmcharter** [comment]
Humour one last challenge and then I'll drop this.

Imagine I have a long-standing community that I don't want rid of. But I have one inactive lead mod, and a rogue mod who's decided to vandalise the community.

Clearly, I would remove the rogue mod asap. But assuming I can't get in touch with the inactive lead mod, and nobody else wants to step up to run the community, what can I do? In this situation, I can't even add myself as a mod to take over in the interim.

**Comment by dessalines** [comment]
You can appoint someone else as mod of that community, transfer it to them, and they can do it.

**Comment by jmcharter** [comment]
Yes, but in the scenario I just suggested (a real example), I'm saying there are no other volunteers to be a mod. The only person willing to take it on, at least in the interim, is me, the admin.
